Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.py

```diff
@@ -1412,10 +1542,16 @@ def get_endpoints_command(client, args):
             sort_by_first_seen=sort_by_first_seen,
             sort_by_last_seen=sort_by_last_seen
         )
+    context = {
+        f'{INTEGRATION_CONTEXT_BRAND}.Endpoint(val.endpoint_id == obj.endpoint_id)': endpoints,
+        'Endpoint(val.ID == obj.ID)': return_endpoint_standard_context(endpoints)
+    }
+    account_context = create_account_context(endpoints)
+    if account_context:
```

Comment:
why do you have a check here and not on [here (line 1903)](https://github.com/demisto/content/pull/10627/files#diff-d3ec87f48a386d9632301be52a63780241c5103d4d43e80919e048c26cdd454eR1903)